test/Cli/Func.TestFramework/Commands/FuncNewCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Xunit.Abstractions;
+
+namespace Func.TestFramework.Commands
+{
+    public class FuncNewCommand : FuncCommand
+    {
+        private string _funcPath;
+        private string _testName;
+
+        public FuncNewCommand(string funcPath, string testName, ITestOutputHelper log) : base(log)
+        {
+            _funcPath = funcPath;
+            _testName = testName;
+
```

Comment:
```suggestion
```

Please review all of the style across all files to fix these nits. We should get style cop setup soon so we can avoid these kinds of reviews